### DISCUSSION TITLE
I want to unlock a banned user.

### DISCUSSION THREAD

**Discussion by k-toki** [danny-avila/LibreChat#3593] [Q&A]
I would like to enable users who cannot log in with the message
”Your account has been temporarily banned due to violations of our service.”

I have a fixed IP address in my company network, and I would like to ease the restrictions on the function that detects a large number of users from a single IP address and bans them for a certain period of time.

**Comment by danny-avila** [comment]
https://github.com/danny-avila/LibreChat/discussions?discussions_q=is%3Aopen+banned

**Reply by danny-avila** [reply]
https://www.librechat.ai/docs/configuration/mod_system
